pkg/virt-api/rest/subresource_test.go

```diff
@@ -947,13 +942,17 @@ var _ = Describe("VirtualMachineInstance Subresources", func() {
 	Context("Pausing", func() {
 		It("Should pause a running, not paused VMI", func() {
 
+			backend.AppendHandlers(
+				ghttp.CombineHandlers(
```

Comment:
duplicate with line 104ff?

Reply:
fixed.

Reply:
I still see the same handler at line 106?

Reply:
now really gone.